CollapseLauncher/Program.cs

```diff
@@ -65,6 +65,9 @@ public static void Main(params string[] args)
                 FileVersionInfo winappSDKver = FileVersionInfo.GetVersionInfo("Microsoft.ui.xaml.dll");
                 LogWriteLine(string.Format("Runtime: {0} - WindowsAppSDK {1}", RuntimeInformation.FrameworkDescription, winappSDKver.ProductVersion), LogType.Scheme, true);
 
+                using (Process collapseProcess = Process.GetCurrentProcess())
+                    collapseProcess.PriorityBoostEnabled = true;
```

Comment:
This can be simplified to `(Process.GetCurrentProcess().PriorityBoostEnabled = true)`

Reply:
Resolved in https://github.com/neon-nyan/Collapse/pull/248/commits/95a34bc80ba31119a10e514e52e6a37c54fed894